### DISCUSSION TITLE
Set the item to 'Do not place other items on top.'

### DISCUSSION THREAD

**Discussion by radamich** [dvdoug/BoxPacker#613] [Q&A] [answered]
Hello Doug,

I have some products where I need to prevent other items from being placed on top. Is there an easy solution for this?
Do you plan to implement this into the project?

Thank you very much.

**Comment by dvdoug** [comment, marked as answer]
Hi @radamich 

There's an example in the docs that implements this that you should hopefully be able to adapt, see https://boxpacker.io/en/stable/custom-constraints.html#example-don-t-allow-batteries-to-be-stacked

**Reply by radamich** [reply]
Thank you so much for your answer. I missed the example in the docs. Great! Thank you.
